application/domain/domain.go

```diff
@@ -24,14 +26,30 @@ const (
 	EmptyUUID                  = "00000000-0000-0000-0000-000000000000"
 	DateFormat                 = "2006-01-02"
 	MaxFileSize                = 1 << 20 // 1 Mebibyte
-	UIURLEnv                   = "UI_URL"
 	AccessTokenLifetimeSeconds = 3600
+	DateTimeFormat             = "2006-01-02 15:04:05"
+
+	// Environment Variables
```

Comment:
Good groupings. I've seen this done with new `const` blocks. Thoughts on that?

Reply:
Sounds good.